qemu/tests/rng_maxbytes_period.py

```diff
@@ -74,7 +74,7 @@ def _is_rngd_running():
         if s:
             test.error(o)
         test.log.info(o)
-        data_rate = re.search(r'\s(\d+\.\d+) kB/s', o, re.M)
+        data_rate = re.search(r'\s(\d+\.\d+)|(\d+) kB/s', o, re.M)
```

Comment:
In this test case, the data rate should be less than 2kB/s, so the pattern "\d+\.\d+" looks make sense. But with your change, it can capture all data rates like `10 kB/s` or `1.1 kB/s`, it's also okay for me. What do you think about this?

Reply:
@BohdanMar @PaulYuuu As pointed out in the bug comment, the issue might be also caused by the s390-trng device, so Bogdan can try to replace it with virtio-rng device and then repeat the test.

Reply:
@ngu-niny Got it, I will try and paste the results here if it depends on s390-trng device